pkg/api/controller.go

```diff
@@ -17,6 +18,10 @@ import (
 	"github.com/gorilla/mux"
 )
 
+const (
+	ioTimeout = 15
```

Comment:
call this idleTimeout and set it to the full timeout value
IdleTimeout =   120 * time.Second,